HelpSource/Classes/Pipe.schelp

```diff
@@ -37,7 +37,8 @@ returns:: The exit status of the command (an Integer).
 Examples::
 
 note::
-For anyone still using OS X 10.3, UNIX commands like pipe do not work when the server is booted; quit the server, otherwise SuperCollider crashes. More recent OS X is not affected.
+For anyone still using macOS 10.3, unix commands like pipe do not work when the server is booted; quit the server, otherwise sc crashes. More recent macOS is not affected.
```

Comment:
Ok! I'm still learning the more advanced features of git so I have no idea what's easiest.